www/searchutil.php

```diff
@@ -568,6 +570,30 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
                 }
                 break;
 
+            case 'rounded_median_time_in_minutes':
+            case 'rounded_median_time_in_minutes/60':
+                // we need to join the gametimes mv table for this query    
```

Comment:
If we wanted to do `3h27m`, we'd do it here. Here, we'd write a "parser" for the `$txt`.

I think the most straightforward way is to do something like this:
* `explode("-", $txt)`, and count the parts, to see if there's a min+max, or just a min, just a max, or just a straight value
* For each part, convert it into a number of minutes, with an `if`/`else if` chain, doing `preg_match` for: `/^([0-9]+)h(\d+)m$/`, then `/^([0-9]+)h$/`, then `/^([0-9]+)m$/`, then `/^([0-9+)$/` (which we'll assume to mean minutes).
* Finally, construct the `$expr` based on min/max/exact `$minutes`.

Reply:
Do you want to do that?

Reply:
I'm open to that, but not this week. (I was telling @salty-horse the other day that I've been spending a bit too much time on IFDB lately…)